src/components/Trilemma/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react"
+import {
+  Drawer,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerOverlay,
+  Flex,
+  Heading,
+  Hide,
+  Show,
+  Text,
+} from "@chakra-ui/react"
+import Card from "../Card"
+import Translation from "../Translation"
+import { TriangleSVG, IProps as TriangleSVGProps } from "./Triangle"
+import { useTrilemma } from "./use-trilemma"
+
+export interface IProps {}
+
+const Trilemma: React.FC<IProps> = () => {
+  const {
+    trilemmaChecks,
+    mobileModalOpen,
+    handleClick,
+    handleModalClose,
+    cardDetail,
+  } = useTrilemma()
+
+  const triangleSVGProps: TriangleSVGProps = {
+    handleClick,
+    ...trilemmaChecks,
+  }
+
+  return (
+    <Flex
+      alignItems="flex-start"
+      flexDirection={{ base: "column", lg: "row" }}
+      justifyContent="space-between"
+    >
+      <Flex
+        flexDirection="column"
+        my={{ base: 8, md: 12 }}
+        mx={{ md: 12 }}
+        py={8}
+        flex={{
+          base: "1 1 100%",
+          lg: "0 1 500px",
+        }}
+      >
+        <Heading fontSize="2rem" mt={0}>
+          <Translation id="page-upgrades-vision-trilemma-h2" />
+        </Heading>
+        <Text>
+          <Translation id="page-upgrades-vision-trilemma-p" />
+        </Text>
+        <Text>
+          <Translation id="page-upgrades-vision-trilemma-p-1" />
+        </Text>
+        <Text>
+          <Translation id="page-upgrades-vision-trilemma-p-2" />
+        </Text>
+        {/* TODO: replace `l` with `lg` after UI migration */}
+        <Show below="l">
```

Comment:
Can you remind me where this breakpoint is coming from? I don't think we are defining the `l` breakpoint in the chakra theme.

Reply:
@pettinarip I believe there is still a conflict with the original implementation of styled-components and emotion, where the theming sometimes gets pulled from the older config that still has to be applied.

I've only seen this issue really occur with the media query components because of the hooks that are called under the hood to retrieve the theme vals.

Reply:
Right. Thanks for the reminder.

I was thinking that we can just start using the `lg` value here since:
- on the parent component we are already using the chakra breakpoints `flexDirection={{ base: "column", lg: "row" }}` 
- the impact of this new value is only going to be inside this component. This will not have a negative UX in any other component

Reply:
@pettinarip right. A weird behavior!

What I should do then is keep implementing Chakra's `useToken` to extract the value from the Chakra theme and apply to the component, guaranteeing no breaking changes on migration completion (adding a TODO to remove later)

For consistency, there might be a couple of other migrated components that would need this update.

What do you think?

Reply:
That's perfect, yes 💪🏼

Reply:
Ok!

I'll make the change here, and if there are still others that need that change I'll create a new PR. 😁